libkbfs/tlf_journal.go

```diff
@@ -1902,19 +1902,19 @@ func (j *tlfJournal) getMDRange(
 }
 
 func (j *tlfJournal) doPutMD(ctx context.Context, rmd *RootMetadata,
-	mdInfo unflushedPathMDInfo,
-	perRevMap unflushedPathsPerRevMap) (
-	mdID kbfsmd.ID, retryPut bool, err error) {
+	mdInfo unflushedPathMDInfo, perRevMap unflushedPathsPerRevMap,
+	verifyingKey kbfscrypto.VerifyingKey) (
```

Comment:
Ah, I see now the `lastWriterVerifyingKey` in `folderBranchOps`. Hmm. I guess leave it then, although a comment somewhere would be good. But where?